### ISSUE TITLE
Follow HTTP/2 best practices

### ISSUE THREAD

**Issue by Alanaktion** [svlt/front#10]
HTTP/2 is awesome, and browser support is already pretty good. Our browser support is already [quite limited](http://caniuse.com/#search=crypto) due to the crypto and worker requirements, and most browsers that support our base application-level requirements also [at least partially support HTTP/2](http://caniuse.com/#search=http2). Because of this, we shouldn't optimize the site for HTTP 1.1, we should take advantage of HTTP/2's parallel asset loading and connection re-use.

Basically we should keep minifying, but stop concatenating. Defering loading of extra assets via custom JS should be avoided, we can use `async` instead for most things and HTTP/2 will load them in parallel, so there's no real delay in most cases.

There are plenty of comprehensive articles detailing the differences for development, here are a few:
- https://blog.cloudflare.com/http-2-for-web-developers/
- https://www.smashingmagazine.com/2016/02/getting-ready-for-http2/
